Formula/zchunk.rb

```diff
@@ -0,0 +1,29 @@
+class Zchunk < Formula
+  desc "Compressed file format for efficient deltas"
+  homepage "https://github.com/zchunk/zchunk"
+  url "https://github.com/zchunk/zchunk/archive/refs/tags/1.3.0.tar.gz"
+  sha256 "5563baa254b256e30e1fea87f94f53a5cf63a074898644f3f7ae5b58f446db03"
+  license "BSD-2-Clause"
+  head "https://github.com/zchunk/zchunk.git", branch: "main"
+
+  depends_on "argp-standalone" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "openssl@1.1"
+  depends_on "zstd"
+
+  uses_from_macos "curl"
+
+  def install
+    system "meson", "setup", "build", *std_meson_args
+    system "meson", "compile", "-C", "build", "--verbose"
+    system "meson", "install", "-C", "build"
+  end
+
+  test do
+    system bin/"zck", test_fixtures("test.png")
+    system bin/"unzck", testpath/"test.png.zck"
+    assert_equal Digest::SHA256.hexdigest(test_fixtures("test.png").read),
+                 Digest::SHA256.hexdigest((testpath/"test.png").read)
```

Comment:
Since we're already reading both files into memory, why not just compare the files directly?